src/libinjector/win_injector.c

```diff
@@ -1350,7 +1366,6 @@ static event_response_t injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_info_t*
                 break;
             default:
                 // TODO Implement
-                success = false;
```

Comment:
?

Reply:
See the statement before switch.

`bool success = false;`